src/cljam/pileup/pileup.clj

```diff
@@ -36,7 +36,8 @@
         count-fn (fn [xs]
                    (if (= n-threads 1)
                      (map (fn [[start end]]
-                            (count-for-positions (read-fn rdr start end) start end)) xs)
+                            (with-open [r (bam/clone-reader rdr)]
```

Comment:
We don't need to clone the reader because it'll be accessed from only one thread.